<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DicedChip_ChipTestForm</class>
 <widget class="QWidget" name="DicedChip_ChipTestForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="slotsGridLayout"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="chipSlotLabel">
     <property name="text">
      <string>No active slot</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="chipFoundLabel">
     <property name="text">
      <string>Finding chip</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="skipPushButton">
     <property name="text">
      <string>Skip Chip</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="confirmPushButton">
     <property name="text">
      <string>Confirm Alignment</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="redoPushButton">
     <property name="text">
      <string>Redo Alignment</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="testPushButton">
     <property name="text">
      <string>Run Chip Test</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="nextPushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Test Next Chip</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
